src/openloco/ui/dropdown.cpp

```diff
@@ -5,13 +5,26 @@ using namespace openloco::interop;
 
 namespace openloco::ui::dropdown
 {
+    static loco_global<int16_t, 0x0113D84E> _dropdownHighlightedIndex;
     static loco_global<string_id[40], 0x0113D850> _dropdownItemFormats;
+    static loco_global<string_id[40], 0x0113D8A0> _dropdownItemArgs;
```

Comment:
There's 8 bytes of 'format args' for each item. `string_id[40][4]` could work as a stopgap solution